Directory.Build.targets

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do you need a `Directory.Build.targets` if it's empty?

Reply:
That's a good question. We had it in core Autofac so I just brought it over here, but... let me test. I could be just cargo-cult-copy/pasting here.

Reply:
Turns out it's not necessary. I'll nuke it.